multipoint/utils.py

```diff
@@ -4,6 +4,7 @@
 import copy
 from mpi4py import MPI
 import numpy as np
+from baseclasses.utils import Error
```

Comment:
Why do we need this additional import here?

Reply:
The `createGroups` function in there uses it.

Reply:
I see, I got confused by the single commit